### PR TITLE
Convert kubernetes-e2e-upgrades-gce-cross-image-and-gci of 1.3

### PR DESCRIPTION
**Pull request by xiangpengzhao**
#1230

It may conflict with #1847 in some files. Will rebase when it happens.

cc @krzyzacy @fejta

### CODE REVIEW

File: jenkins/bootstrap_test.py [hunk 1]
```diff
@@ -1268,7 +1268,16 @@ def testBootstrapCIYaml(self):
             'kubernetes-e2e-gke-1.3-1.4-upgrade-master',
             'kubernetes-e2e-gce-1.3-1.4-upgrade-master',
             'kubernetes-e2e-gce-1.3-1.4-upgrade-cluster',
-            'kubernetes-e2e-gce-1.3-1.4-upgrade-cluster-new'
+            'kubernetes-e2e-gce-1.3-1.4-upgrade-cluster-new',
```
Comment (xiangpengzhao) [inline jenkins/bootstrap_test.py]: Will remove these gce upgrade jobs here after all of them have been merged, and modify the blacklist pattern.

**Comment by krzyzacy** [comment]
I merged the previous one, please rebase :-)

File: jenkins/bootstrap_test.py (hunk 1 above)
Comment (krzyzacy) [inline jenkins/bootstrap_test.py]: sgtm

**Comment by xiangpengzhao** [comment]
Rebased. PTAL. @krzyzacy

**Comment by krzyzacy** [comment]
@k8s-bot bazel test this

File: jobs/config.json [hunk 2]
```diff
@@ -2259,6 +2259,78 @@
...
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-kubernetes-e2e-gce-gci-latest-1.3-gci-latest-upgrade-master.env"
+  ]
+},
+
+"ci-kubernetes-e2e-gce-gci-latest-1.3-gci-latest-upgrade-cluster": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-kubernetes-e2e-gce-gci-latest-1.3-gci-latest-upgrade-clusterr.env"
```
Comment (krzyzacy) [inline jobs/config.json]: clusterr :-)

Comment (xiangpengzhao) [inline jobs/config.json]: Oops, so sorry... :sweat_smile:  Good catch! Will address it.

**Comment by xiangpengzhao** [comment]
Fixed :) PTAL. @krzyzacy

**Comment by krzyzacy** [comment]
@spxtr seems I need to manually trigger the test everytime the PR is updated, even if I have `ok to test`ed?

**Comment by krzyzacy** [comment]
@k8s-bot bazel test this

**Pull request merged**